cloudevents/http/event.py

```diff
@@ -77,6 +77,15 @@ def __eq__(self, other):
     def __getitem__(self, key):
         return self._attributes[key]
 
+    def get(self, key: str, default: typing.Optional[typing.Any] = None) -> typing.Optional[typing.Any]:
+        """
+        Get a an attribute value, return default if does not exist.
+        MUST NOT throw exception when the key does not exist
+        :param key: attribute name
+        :param default: value which will be returned if the attribute does not exist
```

Comment:
```suggestion
        Retrieves an event attribute value for the given key. Returns the default value if not attribute for the given key exists.
        
        MUST NOT throw an exception when the key does not exist.
        
        :param key: The event attribute name.
        :param default: The default value to be returned when no attribute with the given key exists.
```
Can you please also document the returned value?

Reply:
@sasha-tkachev you've not accepted the suggestion nor fixed the grammar. Please take a look a this one once again.

Reply:
@xSAVIKx  Sorry, completely missed the suggested change.